engine/test-src/org/pentaho/di/job/entries/getpop/JobEntryGetPOPTest.java

```diff
@@ -64,16 +64,16 @@ public void before() throws IOException, KettleException, MessagingException {
 
     Mockito.when( parentJob.getLogLevel() ).thenReturn( LogLevel.BASIC );
     entry.setParentJob( parentJob );
-    entry.setSaveMessage( true );
+    entry.setSaveMessage(true);
```

Comment:
This (and many other changes) violate the checkstyle rules for this project - please see my comment on PDI-14305.